### ISSUE TITLE
Bash scripts? | Sværhedsgrad?

### ISSUE THREAD

**Issue by scheldejonas** [CphBusCosSem2/week-4-Operating-systems#4]
Er det nyttigt til det her semester at dykke ned i og kunne loops og conditions i bash scripting, så det sidder på rygraden?

**Comment by kaspercphbusiness** [comment]
Nej, det vil ikke være centralt. Det primære formål med bash-scripts er at vide at de findes og have set lidt på hvordan se ser ud sådan at man kan læse dem hvis der opstår behov for det.
